exec.bash

```diff
@@ -94,7 +95,7 @@ CONTAINER_NAME=${CONTAINER_NAME:="${ROOT_DIR##*/}-$EXECMODE-$FOLDER_MD5"}
 CONTAINER_ID_FILENAME=$ROOT_DIR/$EXECMODE-container_id.txt
 
 $PRINT_INFO
-$PRINT_INFO -e "\e[32musing ${IMAGE_NAME%:*}:\e[4;33m${IMAGE_NAME#*:}\e[0m"
+$PRINT_INFO -e "\e[32musing ${IMAGE_NAME%:*}:\e[4;33m${IMAGE_NAME##*:}\e[0m"
```

Comment:
Is this on purpose? If so can you please explain? To my understanding this will basically print the IMAGE_NAME twice and not the EXECUTION mode anymore.

Reply:
The second # makes the script search for the last occurence of ":" to switch color
I had a name that contained a port in the repository URL, the old behavior printed the image name twice.